backend/src/main/java/com/exadel/carinsurance/model/auth/UserResponseEntity.java

```diff
@@ -0,0 +1,30 @@
+package com.exadel.carinsurance.model.auth;
+
+import com.exadel.carinsurance.model.RoleEntity;
+import com.exadel.carinsurance.model.UserEntity;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class UserResponseEntity {
+  private Long userId;
+  private String firstName;
+  private String lastName;
+  private String email;
+  private RoleEntity role;
+
+  public static UserResponseEntity fromUserEntity( UserEntity userEntity ) {
+    return UserResponseEntity.builder()
+        .userId( userEntity.getUserId() )
+        .firstName( userEntity.getFirstName() )
+        .lastName( userEntity.getLastName() )
+        .email( userEntity.getEmail() )
+        .role( userEntity.getRole() )
+        .build();
+  }

```

Comment:
[Optional] Usually it is not advised to have any logic in your POJO\Entity objects as they should be used only for data transfer. Usually it is advised to provide separate Mapper classes that will handle the mapping for you.

Reply:
Done. Created a mapper class